CONTRIBUTING.md

```diff
@@ -36,7 +36,12 @@ you must run a local docker daemon.
 check formatting:
 
     `./gradlew build`
+    
+4. If you are a Windows user, use the alternate command mentioned below to run tests and
+check formatting:
 
+     `gradlew.bat`
```

Comment:
```suggestion
     `gradlew.bat build`
```

Reply:
gradlew.bat build command again does not seem to work out and returns the same error log. So I am reverting the commit back to its original change (gradlew.bat)